app/services/publication-service.js

```diff
@@ -78,4 +78,15 @@ export default class PublicationService extends Service {
     const publicationNumberTakenList = publicationWithId.filter((publicationFlow) => publicationFlow.id !== publicationFlowId);
     return publicationNumberTakenList.toArray().length !== 0;
   }
+
+  async getNewPublicationNextNumber() {
+    const publications = await this.store.query('publication-flow', {
+      sort: '-publication-number',
+    });
+    if (publications !== null) {
+      const latestPublication = publications.toArray()[0];
+      return latestPublication.publicationNumber + 1;
+    }
+    return 0;
+  }
```

Comment:
optioneel: Als je hier 1 resultaat wilt, kan je pageSize van 1 meegeven.
Je krijgt dan wel nog altijd een array denk ik.

Momenteel is het mogelijk om in de sidebar de publicatienummer te wissen, dan heb je daar null.
(edit: dit werkt niet, die code daar is kapot door het nummer veld, zie algemene review comment)
Die query gaat die null mee sorteren, en die staat nu bovenaan
Dus je doet null +1.

![image](https://user-images.githubusercontent.com/22245223/104434936-1f122f80-558c-11eb-82cc-9bb873e73969.png)

Ik denk dat je best een filter bij in die query steekt, vermijden dat je geen null kan saven kan helpen.

Reply:
<img width="257" alt="Screenshot 2021-02-11 at 11 54 09" src="https://user-images.githubusercontent.com/592312/107627635-dff4fe00-6c5f-11eb-9ff6-2d9e8ffed5a7.png">